test/test_owners.csv

```diff
@@ -455,6 +455,10 @@ k8s.io/kubernetes/cmd/kube-proxy/app,luxas,1
 k8s.io/kubernetes/cmd/kubeadm/app/cmd,davidopp,1
 k8s.io/kubernetes/cmd/kubeadm/app/images,saad-ali,1
 k8s.io/kubernetes/cmd/kubeadm/app/util,eparis,1
+k8s.io/kubernetes/cmd/kubeadm/app/cmd,vishh,1
+k8s.io/kubernetes/cmd/kubeadm/app/images,davidopp,1
+k8s.io/kubernetes/cmd/kubeadm/app/preflight,apprenda,0
```

Comment:
@luxas that's automatically added by the script based on the original repo, so I assume you'll see a bunch more of these. If this is a problem, people should fix the script that generates this.